pyrcareworld/pyrcareworld/envs/dressing_env.py

```diff
@@ -0,0 +1,15 @@
+"""Dressing environment for the PhyRC competition."""
+
+from pyrcareworld.envs.base_env import RCareWorld
+from pathlib import Path
+
+
+_TEMPLATE_PATH = Path(__file__).parents[3] / "template"
```

Comment:
For these path, should we consider using some more configurable formats?

Something like:
```
from pathlib import Path
from pyrcareworld.envs.base_env import RCareWorld

class DressingEnv(RCareWorld):
    """Dressing environment for the PhyRC competition."""

    def __init__(self, template_path=None, executable_path=None, *args, **kwargs):
        # Allow paths to be passed as arguments or use defaults
        template_path = Path(template_path or Path(__file__).parents[3] / "template")
        executable_path = Path(executable_path or template_path / "Dressing" / "DressingPlayer.x86_64")
        
        super().__init__(executable_file=str(executable_path), *args, **kwargs)
```